src/main/java/com/google/sps/data/DatastoreManager.java

```diff
@@ -0,0 +1,192 @@
+        List<Long> postIds = (ArrayList<Long>) repEntity.getProperty(Constants.REP_POSTS); 
+        if (postIds == null) {
+            postIds = new ArrayList<>(); 
+        }
+        postIds.add(postId);
+        repEntity.setProperty(Constants.REP_POSTS, postIds); 
+        datastore.put(repEntity);
+    }
+
+    /**
+     * Updates a post entity with a comment in the replies list 
+     * @param postId ID of the post entity 
+     * @param commentId ID of the comment being added as a reply 
+     */ 
+    public static void updatePostWithComment(long postId, long commentId) throws EntityNotFoundException {
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        Entity postEntity = DatastoreManager.queryForPostEntityWithId(postId); 
+        List<Long> commentIds = (ArrayList<Long>) postEntity.getProperty(Constants.POST_REPLIES); 
+        if (commentIds == null) {
+            commentIds = new ArrayList<>(); 
+        }
+        commentIds.add(commentId); 
+        postEntity.setProperty(Constants.POST_REPLIES, commentIds); 
+        datastore.put(postEntity);
+    }
+
+    /**
+     * Updates a post entity with a comment in the replies list 
+     * @param postId ID of the post entity 
+     * @param commentId ID of the comment being added as a reply 
+     */ 
+    public static void updatePostWithAnswer(long postId, long answerId) throws EntityNotFoundException {
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        Entity postEntity = DatastoreManager.queryForPostEntityWithId(postId); 
+        postEntity.setProperty(Constants.POST_ANSWER, answerId); 
+        datastore.put(postEntity);
+    }
+
+    /**
+     * Searches datastore for a particular representative and converts the entity 
+     * to a Representative instance 
+     * @param repName name of the representative to search datastore for 
+     * @return the representative object, or null if it was not found in datastore 
+     */ 
+    public static @Nullable Representative queryForRepresentativeObjectWithName(String repName) {
+        Query query = new Query(Constants.REP_ENTITY_TYPE); 
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        PreparedQuery results = datastore.prepare(query);
+        Entity repEntity = null; 
+        for (Entity entity : results.asIterable()) {
+            String name = (String) entity.getProperty(Constants.REP_NAME);
+            if (name.equals(repName)) {
+                repEntity = entity; 
+            }
+        }
+        if (repEntity == null) {
+            return null; 
+        }
+        else {
+            Representative rep = null; 
+            try {
+                rep = DatastoreEntityToObjectConverter.convertRepresentative(repEntity); 
+                return rep; 
+            } 
+            catch(EntityNotFoundException e) {
+                e.printStackTrace();
```

Comment:
use the logger. not printStrackTrace.

you don't have access to system.out in prod generally.